flowapi/flowapi/user_model.py

```diff
@@ -24,31 +23,39 @@ class UserObject:
     ----------
     username : str
         Name of the user
-    claims : dict
+    scopes : dict
         Dictionary giving a whitelist of the user's claims
```

Comment:
We've never discussed this (I don't think) but do we as an organisation continue to use these terms (black-/whitelist)?